src/views/lpa-details.html

```diff
@@ -34,7 +34,7 @@
     {% endif %}
 
     <h1 class="govuk-heading-l">
-      Which LPA are you submitting data for?
+      Enter LPA details
```

Comment:
no worries about this if you cba, but you could reuse the variable `{{ pageName }}` instead here if you wanted